libnopegl/src/backends/gl/pipeline_gl.c

```diff
@@ -419,10 +419,7 @@ static int pipeline_compute_init(struct pipeline *s)
     struct gpu_ctx_gl *gpu_ctx_gl = (struct gpu_ctx_gl *)s->gpu_ctx;
     struct glcontext *gl = gpu_ctx_gl->glcontext;
 
-    if (NGLI_HAS_FLAGS(gl->features, NGLI_FEATURE_GL_COMPUTE_SHADER_ALL)) {
-        LOG(ERROR, "context does not support compute shaders");
-        return NGL_ERROR_GRAPHICS_UNSUPPORTED;
-    }
+    ngli_assert(NGLI_HAS_FLAGS(gl->features, NGLI_FEATURE_GL_COMPUTE_SHADER_ALL));
```

Comment:
A few more candidates in case you're interested:
```
libnopegl/src/android_ctx.c:        return ((gl->features & features) == features);
libnopegl/src/backends/gl/gpu_ctx_gl.c:        if ((gl->features & feature_gl) == feature_gl)
libnopegl/src/backends/vk/vkcontext.c:            if ((props.queueFlags & flags) == flags)
libnopegl/src/backends/vk/vkcontext.c:            (properties.linearTilingFeatures & features) == features)
libnopegl/src/backends/vk/vkcontext.c:            (properties.optimalTilingFeatures & features) == features)
libnopegl/src/backends/vk/vkcontext.c:        if ((type & (1 << i)) && (s->phydev_mem_props.memoryTypes[i].propertyFlags & props) == props)
libnopegl/src/gen_doc.c:        if ((par->flags & invalid_flags) == invalid_flags) {
libnopegl/src/vaapi_ctx.c:        if ((gl->features & features) == features)
```

Reply:
Done in separate commits.